go/porcelain/deploy.go

```
@@ -263,7 +267,13 @@ func (n *Netlify) DoDeploy(ctx context.Context, options *DeployOptions, deploy *
 
 	options.files = files
 
-	functions, schedules, functionsConfig, err := bundle(ctx, options.FunctionsDir, options.Observer)
+	functionsTmpDir, err := os.MkdirTemp("", "netlify-deploy-functions-")
+	if err != nil {
+		return nil, err
+	}
+	defer os.RemoveAll(functionsTmpDir)
```

golangci-lint (2.12.2)

[error] Error return value of `os.RemoveAll` is not checked

(errcheck)